crates/derive-sqlite/src/lambda.rs

```diff
@@ -186,11 +186,11 @@ fn bind_parameter_node<N: doc::AsNode>(
         Node::PosInt(u) => stmt.raw_bind_parameter(index + 1, u),
         Node::Array(_) => stmt.raw_bind_parameter(
             index + 1,
-            &serde_json::to_string(&doc::SerPolicy::default().on(node)).unwrap(),
+            &serde_json::to_string(&doc::SerPolicy::default().on(node, None)).unwrap(),
```

Comment:
similar to my comment on extract_all -- this function is used a lot, and most call sites don't care about pruning. Rather than making them, how about updating SerPolicy to add additional ways of calling this which take a non-optional bool reference?

Then, this function can be updated to pass a no-op instance without changing its signature.

Reply:
I don't find that objectionable at all, and can update to make "prune-tracking" and "non-tracking" versions. But I'll point out that most of those call sites also use a hard coded `SerPolicy::default().on...`. It feels like trying to improve ergonomics in situations where we don't want to be using a `SerPolicy` in the first place. `default` always returns an unlimited policy, and it doesn't seem like a good idea to change that given the large variety of different call sites (I actually tried and broke things).

In other words, I'm questioning why we're using `SerPolicy` at all if we don't want to actually truncate anything, and also don't care about knowing whether we did or not. It feels to me like we ought to only be truncating data in situations where we're able to report on whether or not we actually did. Some usages, like for extracting keys, seem like they would be better off without a `SerPolicy` at all. Others, like for capture combiners, seem like we'd really want to track whether the policy has actually truncated anything.

Reply:
>  It feels like trying to improve ergonomics in situations where we don't want to be using a SerPolicy in the first place.

That's accurate. When I implemented SerPolicy, I took away the Serialize implementation for the doc::Node trait deliberately, to force each serialization site to think about whether SerPolicy was applicable or not. So, yea, the goal was to have it be as succinct as possible but to force the writer and reader to think about it every time.